src/ui/DynamicFacet/DynamicFacet.ts

```diff
@@ -674,6 +674,10 @@ export class DynamicFacet extends Component implements IAutoLayoutAdjustableInsi
   }
 
   private toggleSearchDisplay() {
+    if (this.options.enableFacetSearch === false) {
```

Comment:
is `this.options.enableFacetSearch` always a boolean?
if yes, you could do 
```
if (!this.options.enableFacetSearch) {
    return;
}
```

Reply:
no sometimes it is undefined

Reply:
It would work anyways but ok 😄

Reply:
I clarified with Jean-Felix, the `undefined` case has a special behaviour at the moment that is handled on line 681. If enableFacetSearch is `undefined` and more values are available, we show the search box. Otherwise we do not.

Reply:
oh ok great